src/styles/blocks/card.css

```
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
```

img it's `inline` element, so `margin: 0 auto 40px;` don't wok here. Just make it `block`.

```suggestion
.card__image {
  display: block;
  width: 160px;
  height: 134px;
  margin: 0 auto 40px;
}
```